app/src/main/java/ch/epfl/sweng/favors/main/LoggedInScreen.java

```diff
@@ -85,6 +84,8 @@ public void onPropertyChanged(Observable sender, int propertyId) {
             }
         });
         Database.getInstance().updateFromDb(User.getMain());
+        
+        mFirestore = FirebaseFirestore.getInstance();
```

Comment:
Also here, we should work with Database.getInstance to access to the db as line 86